src/components/TheLanguageSelector.vue

```diff
@@ -41,11 +42,28 @@ const props = defineProps({
   }
 });
 
+const sortedLocales = computed(() => {
+  const currentLang = locale.value;
+  return [ ...availableLocales ].sort((a, b) => {
+    if (a === currentLang){
+      return -1;
+    }
+    if (b === currentLang){
+      return 1;
+    }
+    return a.localeCompare(b);
+  });
+});
```

Comment:
Are we trying to sort all of them? Or just bring the current to the top?

Reply:
@Haviles04 the idea is to put the current lang at the top and then to sort alphabetically for the others. This looks right to me

Reply:
Cool Yea, for some reason I thought I read In the chat we weren't gonna sort the rest.. disregard :) LGTM